### PR TITLE
feat: modif with repo name modification

### PR DESCRIPTION
**Pull request by cowan-macady**
changed old repo name in files where it was present

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This project serves as a Software Development Kit for developers of Indykite applications.
 
-[![codecov](https://codecov.io/gh/indykite/jarvis-sdk-python/branch/master/graph/badge.svg?token=8u4yvyi5PN)](https://codecov.io/gh/indykite/jarvis-sdk-python)
+[![codecov](https://codecov.io/gh/indykite/indykite-sdk-python/branch/master/graph/badge.svg?token=8u4yvyi5PN)](https://codecov.io/gh/indykite/indykite-sdk-python)
```
Comment (LadislavFolta) [inline README.md:5]: Not sure if the token stays the same?

Comment (cowan-macady) [reply README.md:5]: do you know how to change the repo name on codecov.io?

Comment (LadislavFolta) [reply README.md]: unfortunately no. Perhaps @maprokes knows?

Comment (maprokes) [reply README.md]: I think that you can keep the token, just change the name, codecov should pickup the change automatically, if it doesn't please let us know and we can fix it after renaming, it's probably not a big deal anyway

**Pull request merged**